src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -49,18 +51,18 @@
     private final BigQueryPdao bigQueryPdao;
     private final RestHighLevelClient client;
 
-    private final Map<String, String> columnReplacements = Map.of(
-        "biosample_id", "dct:identifier",
-        "donor_id", "prov:wasDerivedFrom",
-        "disease", "TerraCore:hasDisease",
-        "genus_species", "TerraCore:hasOrganismType",
-        "organ", "TerraCore:hasAnatomicalSite",
-        "library_construction_method_text", "TerraCore:hasLibraryPrep",
-        "sex", "TerraCore:hasSex",
-        "project_title", "dct:title",
-        "cell_type", "TerraCore:hasSelectedCellType",
-        "organism_age_unit", "TerraCore:hasAgeUnit"
-    );
+    private final BiMap<String, String> columnReplacements = new ImmutableBiMap.Builder<String, String>()
```

Comment:
Does the diamond operator not work here? 
```suggestion
    private final BiMap<String, String> columnReplacements = new ImmutableBiMap.Builder<>()
```

Reply:
Surprisingly, it doesn't. It also ends up suggesting `<Object,Object>` instead.